dependency-injection/1.0/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Jakarta Depdency Injection 1.0"
+date: 2019-07-27
+summary: "Final for Jakarta EE 8"
+---
+
+Jakarta Dependency Injection specifies a means for obtaining objects in such a way as to maximize reusability, testability and maintainability compared to traditional approaches such as constructors, factories, and service locators (e.g., JNDI).
+
+
```

Comment:
Missing section pointing to spec docs (PDF and HTML), apidocs, and maven coordinates.

Something like this (not sure about the maven coordinates) ...

``` markdown
* [Jakarta Dependency Injection 1.0 Specification Document](./injection-spec-1.0.pdf) (PDF)
* [Jakarta Dependency Injection 1.0 Specification Document](./injection-spec-1.0.html) (HTML)
* [Jakarta Dependency Injection 1.0 Javadoc](./apidocs)
* Maven coordinates
  * [jakarta.inject:jakarta.inject-api:jar:1.0](https://search.maven.org/artifact/jakarta.inject/jakarta.inject-api/1.0/jar)
```